elephant/statistics.py

```diff
@@ -240,3 +242,276 @@ def lv(v):
     # calculate LV and return result
     # raise error if input is multi-dimensional
     return 3.*np.mean(np.power(np.diff(v)/(v[:-1] + v[1:]), 2))
+
+
+#adaptation to output neo.AnalogSignal and wrapper make_kernel() in
+#instantaneous_rate()
+def make_kernel(form, sigma, resolution, direction=1):
+    """
+    Creates kernel functions for convolution.
+
+    Constructs a numeric linear convolution kernel of basic shape to be used
+    for data smoothing (linear low pass filtering) and firing rate estimation
+    from single trial or trial-averaged spike trains.
+
+    Exponential and alpha kernels may also be used to represent postynaptic
+    currents / potentials in a linear (current-based) model.
+
+    Adapted from original script written by Martin P. Nawrot for the
```

Comment:
Need to clarify the license.  Was this ported from MATLAB code, or did you just use the description of the algorithm?  If ported, what is the license of the MATLAB code?